test.md

```diff
@@ -0,0 +1 @@
+hello
```

Comment:
is this an accident ?

Reply:
Hi, yah, please don’t merge until I’ve confirmed lol. I’m writing code to automatically send pull requests. But, need to test by... sending pull requests